tmt/steps/execute/__init__.py

```diff
@@ -42,15 +49,16 @@ class Execute(tmt.steps.Step):
 
     # Internal executor is the default implementation
     how = 'tmt'
+    data: List[tmt.steps.StepData]
 
-    def __init__(self, plan, data):
+    def __init__(self, plan: "tmt.Plan", data: tmt.steps.StepData) -> None:
         """ Initialize execute step data """
         super().__init__(plan=plan, data=data)
         # List of Result() objects representing test results
-        self._results = []
+        self._results: List[tmt.Result] = []
 
         # List of scripts to install
-        self.scripts = []
+        self.scripts: List[str] = []
```

Comment:
I believe here the `self.scripts` is misplaced, I guess that in 8fcd656c @thrix meant to add this to `ExecutePlugin` not the `Execute` step class.

Reply:
+1, seems this was a mistake

Reply:
Removed.